src/DataProbePostProcessing.C

```diff
@@ -394,6 +404,13 @@ DataProbePostProcessing::load(
 	    else
 	      probeInfo->offsetSpacings_[iplane+offset].push_back(0.0);
 
+            // string: onlyOutputField
+            const YAML::Node onlyOutputField = y_planenode["only_output_field"];
+	    if (onlyOutputField)
```

Comment:
whitespace

Reply:
Not sure what happened with the whitespace there, but it should look better now, at least on my branch.